apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/preadj/R4ClaimResourceProviderIT.java

```diff
@@ -74,11 +72,10 @@ public void shouldGetCorrectClaimResourcesByMbiHash() {
             .search()
             .forResource(Claim.class)
             .where(
-                ImmutableMap.of(
-                    "mbi", Collections.singletonList(new ReferenceParam("a7f8e93f09")),
+                Map.of(
+                    "mbi", List.of(new ReferenceParam("a7f8e93f09")),
                     "service-date",
-                        Arrays.asList(
-                            new DateParam("gt1970-07-18"), new DateParam("lt1970-07-30"))))
+                        List.of(new DateParam("gt1970-07-18"), new DateParam("lt1970-07-30"))))
```

Comment:
How did the transformers go through what looks like such a large change and our IT test have only such small changes?

Reply:
The ITs just look at the final output JSON, so the changes were mostly just updating the expected JSON output files.

Reply:
So this PR was mostly just changing the implementation, and not the output. makes sense!